images/charmed-spark/pebble/log-layer.yaml

```diff
@@ -0,0 +1,13 @@
+log-targets:
+  grafana-agent-k8s:
+    override: replace
+    type: loki
+    location: $LOKI_URL
+    services: [all]
+    labels:
+      product: charmed-spark
+      role: $FLAVOUR
+      app: spark
+      spark_job_id: $SPARK_APPLICATION_ID
+      user: $SPARK_USER
+      hostname: $HOSTNAME
```

Comment:
**niptick, suggestion** maybe `pod` is more appropriate here? Anyhow, it is not a strong major point. Up to you

Reply:
I believe `pod` is more appropriated in this context at the end what it is contained is the pod name.

Reply:
Done.